packages/xdl/src/Webpack.ts

```diff
@@ -363,3 +415,133 @@ async function getWebpackConfigEnvFromBundlingOptionsAsync(
     ...(options.webpackEnv || {}),
   };
 }
+
+async function startNextJsAsync({ projectRoot, port, dev, expoConfig, onFinished }) {
+  let next;
+  try {
+    const { exp } = await ProjectUtils.readConfigJsonAsync(projectRoot);
```

Comment:
```suggestion
    const { exp } = await ConfigUtils.readConfigJsonAsync(projectRoot);
```
(`ProjectUtils.readConfigJsonAsync` will be deprecated soon.)